### PR TITLE
feat: api for candidates and parties + db population script

### PR DESCRIPTION
**Pull request by Jatewo**
## Description

Adds routes for getting candidates and parties and a script for populating the database with mockdata. 

## Changes made
- Added GET routes for candidates and parties
- Setup of nodemon
- Added workflows for test and linting
- Added cors settings
- Added temporary routes for voting
- Added submodules
- Setup of MongoDB/Mongoose

### CODE REVIEW

**Review by TheMOHD3886 — APPROVED** [review]
10/10 bro, looks good. Could use more a few extra tests next time, but I'll approve anyway 😉

File: package.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  },
+  "devDependencies": {
+    "@eslint/js": "^9.11.1",
+    "eslint": "^9.11.1",
+    "eslint-plugin-jest": "^28.8.3",
+    "eslint-plugin-jsdoc": "^50.4.3",
+    "globals": "^15.9.0",
+    "npm-run-all": "^4.1.5",
+    "pg": "^8.13.0",
+    "sequelize": "^6.37.3",
```
Comment (MortenMunk) [inline package.json]: uninstall sequelize

**Review by MortenMunk — CHANGES_REQUESTED** [review]
We dont use sequelize anymore

**Review by MortenMunk — APPROVED** [review]
lgtm

**Pull request merged**